.circleci/config.yml

```diff
@@ -202,6 +206,8 @@ jobs:
           name: stack-size
           command: |
             make npm-link
+            pip install ./pyodide-test-runner
+            npm install -g node-fetch@2
```

Comment:
Why do we need to install node-fetch? It should already be installed I think?

Reply:
node-fetch is installed locally during the installation of pyodide JS package, which means it cannot be accessed unless the js script file (`node_test_driver.js`) is located in the subdirectory of Pyodide source tree. In this PR we are installing `pyodide-test-runner`  to e.g. site-packages, so node-fetch needs to be installed globally.

Maybe we can update the node version in our docker image in a subsequent PR that supports native fetch to remove those duplicated node-fetch installations.